WindDirectionControl/WindDirectionControl.module.less

```diff
@@ -0,0 +1,29 @@
+// WindDirectionControl.module.less
+//
+@import '../styles/mixins.less';
+@import '../styles/skin.less';
+@import '../styles/variables.less';
+
+.windDirectionControl {
```

Comment:
please add the 320px width/height, according to specifications + center the svgs vertically and horizontally inside the parent div